README.rst

```diff
@@ -0,0 +1,219 @@
+
+.. code-block:: console
+
+    tarantoolctl rocks install cartridge-cli
+    .rocks/bin/cartridge create --name myapp
+    cd myapp
+    ../.rocks/bin/cartridge build
+    ../.rocks/bin/cartridge start
+
+That's all! Now you can visit http://localhost:8081 and see your application's
+Admin Web UI:
+
+.. image:: https://user-images.githubusercontent.com/11336358/75786427-52820c00-5d76-11ea-93a4-309623bda70f.png
+   :align: center
+   :scale: 100%
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Next steps
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+See:
+
+* A more detailed
+  `getting started guide <https://www.tarantool.io/en/doc/2.2/getting_started/getting_started_cartridge/>`_
+* More
+  `application examples <https://github.com/tarantool/examples>`_
+* `Cartridge documentation <https://www.tarantool.io/en/doc/latest/book/cartridge/>`_
+* `Cartridge API reference <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_api/>`_
+
+--------------------------------------------------------------------------------
+Contribution
+--------------------------------------------------------------------------------
+
+The workflow for Cartridge contributors may be different from that for Cartridge
+users as it it implies building the project from source (documentation, Web UI)
+and running tests.
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Building from source
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Prerequisites:
+
+* ``tarantool``, ``tarantool-dev`` (see these `instructions <https://www.tarantool.io/en/download/?v=1.10>`_);
+* ``git``, ``gcc``, ``cmake``.
+
+The fastest way to build the project is to skip building the Web UI:
+
+.. code-block:: console
+
+    CMAKE_DUMMY_WEBUI=true tarantoolctl rocks make
+
+But if you want to build the frontend too, you'll also need:
+
+* ``nodejs`` >= 8 (see these `instructions <https://github.com/nodesource/distributions>`_;
+* ``npm`` >= 6.
+
+Documentation is generated from source code, but only if the ``ldoc`` and ``sphinx``
+tools are installed:
+
+.. code-block:: console
+
+    pip install 'sphinx==3.0.3'
+    tarantoolctl rocks install \
+      https://raw.githubusercontent.com/tarantool/LDoc/tarantool/ldoc-scm-2.rockspec \
+      --server=http://rocks.moonscript.org
+    tarantoolctl rocks make
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Running a demo cluster
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There are several example entry points which are mostly used for testing,
+but can also be useful for demo purposes or experiments:
+
+.. code-block:: console
+
+    tarantoolctl rocks install cartridge-cli
+    .rocks/bin/cartridge start
+
+    # or select a specific entry point
+    # .rocks/bin/cartridge start --script ./test/entrypoint/srv_basic.lua
```

Comment:
It's not `console` syntax, but a shell script.
Otherwise colors are inverted: code is gray, and comments are colored

![image](https://user-images.githubusercontent.com/2205188/83871577-fb0a8800-a738-11ea-97d8-237241c6a0fb.png)

Reply:
Postponed to the future: https://github.com/tarantool/cartridge/issues/885